src/dune_util/value.mli

```diff
@@ -13,6 +13,10 @@ val to_string : t -> dir:Path.t -> string
 
 val to_path : ?error_loc:Loc.t -> t -> dir:Path.t -> Path.t
 
+(** The value corresponds to a dependency of the user it is relative to the
```

Comment:
I couldn't quite parse this comment, maybe I should retry in the morning :)

Reply:
Indeed. I would prefer a description that relies on examples a little more.